22:41:49.864 INFO  main GatewayStart         : config conf.yaml !!ibgroup.web.core.clientportal.gw.Config
authDelay: 3000
ccp: false
cors: {origin.allowed: '*', allowCredentials: false}
ip2loc: US
ips:
  allow: [192.*, 131.216.*, 127.0.0.1]
  deny: [212.90.324.10]
listenPort: 7000
listenSsl: true
portalBaseURL: ''
proxyRemoteHost: https://api.ibkr.com
proxyRemoteSsl: true
sslCert: vertx.jks
sslPwd: mywebapi
ssoPing: 5
svcEnvironment: v1
tst: false
twsBaseURL: /tws.proxy
webApps:
- {cache: true, index: index.html, listing: false, name: demo, proxy: ''}

22:41:50.343 INFO  vert.x-eventloop-thread-0 GatewayHttpProxy     :  -> mount demo on /demo
22:41:50.376 INFO  vert.x-eventloop-thread-0 GatewayVerticle      : version: 65f1ff1d961c97a69e84261ebde3d1940ac72275 Fri, 28 May 2021 09:59:56 -0400
22:41:50.376 INFO  vert.x-eventloop-thread-0 GatewayVerticle      : Java Version: 1.8.0_201
22:41:58.849 WARN  vert.x-eventloop-thread-0 DefaultChannelPipeline : An exceptionCaught() event was fired, and it reached at the tail of the pipeline. It usually means the last handler in the pipeline did not handle the exception.
io.netty.handler.codec.DecoderException: javax.net.ssl.SSLException: Received fatal alert: certificate_unknown
	at io.netty.handler.codec.ByteToMessageDecoder.callDecode(ByteToMessageDecoder.java:459)
	at io.netty.handler.codec.ByteToMessageDecoder.channelRead(ByteToMessageDecoder.java:265)
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:362)
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:348)
	at io.netty.channel.AbstractChannelHandlerContext.fireChannelRead(AbstractChannelHandlerContext.java:340)
	at io.netty.channel.DefaultChannelPipeline$HeadContext.channelRead(DefaultChannelPipeline.java:1359)
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:362)
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:348)
	at io.netty.channel.DefaultChannelPipeline.fireChannelRead(DefaultChannelPipeline.java:935)
	at io.netty.channel.nio.AbstractNioByteChannel$NioByteUnsafe.read(AbstractNioByteChannel.java:134)
	at io.netty.channel.nio.NioEventLoop.processSelectedKey(NioEventLoop.java:645)
	at io.netty.channel.nio.NioEventLoop.processSelectedKeysOptimized(NioEventLoop.java:580)
	at io.netty.channel.nio.NioEventLoop.processSelectedKeys(NioEventLoop.java:497)
	at io.netty.channel.nio.NioEventLoop.run(NioEventLoop.java:459)
	at io.netty.util.concurrent.SingleThreadEventExecutor$5.run(SingleThreadEventExecutor.java:858)
	at java.lang.Thread.run(Unknown Source)
Caused by: javax.net.ssl.SSLException: Received fatal alert: certificate_unknown
	at sun.security.ssl.Alerts.getSSLException(Unknown Source)
	at sun.security.ssl.SSLEngineImpl.fatal(Unknown Source)
	at sun.security.ssl.SSLEngineImpl.fatal(Unknown Source)
	at sun.security.ssl.SSLEngineImpl.recvAlert(Unknown Source)
	at sun.security.ssl.SSLEngineImpl.readRecord(Unknown Source)
	at sun.security.ssl.SSLEngineImpl.readNetRecord(Unknown Source)
	at sun.security.ssl.SSLEngineImpl.unwrap(Unknown Source)
	at javax.net.ssl.SSLEngine.unwrap(Unknown Source)
	at io.netty.handler.ssl.SslHandler$SslEngineType$3.unwrap(SslHandler.java:281)
	at io.netty.handler.ssl.SslHandler.unwrap(SslHandler.java:1215)
	at io.netty.handler.ssl.SslHandler.decodeJdkCompatible(SslHandler.java:1127)
	at io.netty.handler.ssl.SslHandler.decode(SslHandler.java:1162)
	at io.netty.handler.codec.ByteToMessageDecoder.decodeRemovalReentryProtection(ByteToMessageDecoder.java:489)
	at io.netty.handler.codec.ByteToMessageDecoder.callDecode(ByteToMessageDecoder.java:428)
	... 15 common frames omitted
22:41:58.970 WARN  vert.x-eventloop-thread-0 DefaultChannelPipeline : An exceptionCaught() event was fired, and it reached at the tail of the pipeline. It usually means the last handler in the pipeline did not handle the exception.
io.netty.handler.codec.DecoderException: javax.net.ssl.SSLException: Received fatal alert: certificate_unknown
	at io.netty.handler.codec.ByteToMessageDecoder.callDecode(ByteToMessageDecoder.java:459)
	at io.netty.handler.codec.ByteToMessageDecoder.channelRead(ByteToMessageDecoder.java:265)
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:362)
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:348)
	at io.netty.channel.AbstractChannelHandlerContext.fireChannelRead(AbstractChannelHandlerContext.java:340)
	at io.netty.channel.DefaultChannelPipeline$HeadContext.channelRead(DefaultChannelPipeline.java:1359)
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:362)
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:348)
	at io.netty.channel.DefaultChannelPipeline.fireChannelRead(DefaultChannelPipeline.java:935)
	at io.netty.channel.nio.AbstractNioByteChannel$NioByteUnsafe.read(AbstractNioByteChannel.java:134)
	at io.netty.channel.nio.NioEventLoop.processSelectedKey(NioEventLoop.java:645)
	at io.netty.channel.nio.NioEventLoop.processSelectedKeysOptimized(NioEventLoop.java:580)
	at io.netty.channel.nio.NioEventLoop.processSelectedKeys(NioEventLoop.java:497)
	at io.netty.channel.nio.NioEventLoop.run(NioEventLoop.java:459)
	at io.netty.util.concurrent.SingleThreadEventExecutor$5.run(SingleThreadEventExecutor.java:858)
	at java.lang.Thread.run(Unknown Source)
Caused by: javax.net.ssl.SSLException: Received fatal alert: certificate_unknown
	at sun.security.ssl.Alerts.getSSLException(Unknown Source)
	at sun.security.ssl.SSLEngineImpl.fatal(Unknown Source)
	at sun.security.ssl.SSLEngineImpl.fatal(Unknown Source)
	at sun.security.ssl.SSLEngineImpl.recvAlert(Unknown Source)
	at sun.security.ssl.SSLEngineImpl.readRecord(Unknown Source)
	at sun.security.ssl.SSLEngineImpl.readNetRecord(Unknown Source)
	at sun.security.ssl.SSLEngineImpl.unwrap(Unknown Source)
	at javax.net.ssl.SSLEngine.unwrap(Unknown Source)
	at io.netty.handler.ssl.SslHandler$SslEngineType$3.unwrap(SslHandler.java:281)
	at io.netty.handler.ssl.SslHandler.unwrap(SslHandler.java:1215)
	at io.netty.handler.ssl.SslHandler.decodeJdkCompatible(SslHandler.java:1127)
	at io.netty.handler.ssl.SslHandler.decode(SslHandler.java:1162)
	at io.netty.handler.codec.ByteToMessageDecoder.decodeRemovalReentryProtection(ByteToMessageDecoder.java:489)
	at io.netty.handler.codec.ByteToMessageDecoder.callDecode(ByteToMessageDecoder.java:428)
	... 15 common frames omitted
22:42:00.909 INFO  vert.x-eventloop-thread-0 BaseServiceProxy     : -> GET /sso/Login?forwardTo=22&RL=1&ip2loc=US,200|1653ms
22:42:00.910 INFO  vert.x-eventloop-thread-0 CookieManager        : set cookies [XYZAB_AM.LOGIN=""; Domain=.ibkr.com; Path=/;Secure;SameSite=None, XYZAB=""; Domain=.ibkr.com; Path=/;Secure;SameSite=None, URL_PARAM="forwardTo=22&RL=1&ip2loc=US"; Version=1; Domain=.ibkr.com; Path=/;Secure;HttpOnly, JSESSIONID=CDEE94B76061601ECE67DA5E77B5BBD8.ny5wwwdam1-internet; Path=/sso; HttpOnly;Secure;SameSite=None]
22:42:00.911 DEBUG vert.x-eventloop-thread-0 BaseServiceProxy     : Remapping Set-cookies [XYZAB_AM.LOGIN=""; Domain=.ibkr.com; Path=/;Secure;SameSite=None, XYZAB=""; Domain=.ibkr.com; Path=/;Secure;SameSite=None, URL_PARAM="forwardTo=22&RL=1&ip2loc=US"; Version=1; Domain=.ibkr.com; Path=/;Secure;HttpOnly, JSESSIONID=CDEE94B76061601ECE67DA5E77B5BBD8.ny5wwwdam1-internet; Path=/sso; HttpOnly;Secure;SameSite=None] -> 
22:42:01.192 INFO  vert.x-eventloop-thread-0 BaseServiceProxy     : -> GET /scripts/common/js/jquery-3.4.1/jquery.min.js,200|147ms
22:42:01.323 WARN  vert.x-eventloop-thread-0 DefaultChannelPipeline : An exceptionCaught() event was fired, and it reached at the tail of the pipeline. It usually means the last handler in the pipeline did not handle the exception.
io.netty.handler.codec.DecoderException: javax.net.ssl.SSLException: Received fatal alert: certificate_unknown
	at io.netty.handler.codec.ByteToMessageDecoder.callDecode(ByteToMessageDecoder.java:459)
	at io.netty.handler.codec.ByteToMessageDecoder.channelRead(ByteToMessageDecoder.java:265)
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:362)
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:348)
	at io.netty.channel.AbstractChannelHandlerContext.fireChannelRead(AbstractChannelHandlerContext.java:340)
	at io.netty.channel.DefaultChannelPipeline$HeadContext.channelRead(DefaultChannelPipeline.java:1359)
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:362)
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:348)
	at io.netty.channel.DefaultChannelPipeline.fireChannelRead(DefaultChannelPipeline.java:935)
	at io.netty.channel.nio.AbstractNioByteChannel$NioByteUnsafe.read(AbstractNioByteChannel.java:134)
	at io.netty.channel.nio.NioEventLoop.processSelectedKey(NioEventLoop.java:645)
	at io.netty.channel.nio.NioEventLoop.processSelectedKeysOptimized(NioEventLoop.java:580)
	at io.netty.channel.nio.NioEventLoop.processSelectedKeys(NioEventLoop.java:497)
	at io.netty.channel.nio.NioEventLoop.run(NioEventLoop.java:459)
	at io.netty.util.concurrent.SingleThreadEventExecutor$5.run(SingleThreadEventExecutor.java:858)
	at java.lang.Thread.run(Unknown Source)
Caused by: javax.net.ssl.SSLException: Received fatal alert: certificate_unknown
	at sun.security.ssl.Alerts.getSSLException(Unknown Source)
	at sun.security.ssl.SSLEngineImpl.fatal(Unknown Source)
	at sun.security.ssl.SSLEngineImpl.fatal(Unknown Source)
	at sun.security.ssl.SSLEngineImpl.recvAlert(Unknown Source)
	at sun.security.ssl.SSLEngineImpl.readRecord(Unknown Source)
	at sun.security.ssl.SSLEngineImpl.readNetRecord(Unknown Source)
	at sun.security.ssl.SSLEngineImpl.unwrap(Unknown Source)
	at javax.net.ssl.SSLEngine.unwrap(Unknown Source)
	at io.netty.handler.ssl.SslHandler$SslEngineType$3.unwrap(SslHandler.java:281)
	at io.netty.handler.ssl.SslHandler.unwrap(SslHandler.java:1215)
	at io.netty.handler.ssl.SslHandler.decodeJdkCompatible(SslHandler.java:1127)
	at io.netty.handler.ssl.SslHandler.decode(SslHandler.java:1162)
	at io.netty.handler.codec.ByteToMessageDecoder.decodeRemovalReentryProtection(ByteToMessageDecoder.java:489)
	at io.netty.handler.codec.ByteToMessageDecoder.callDecode(ByteToMessageDecoder.java:428)
	... 15 common frames omitted
22:42:01.324 WARN  vert.x-eventloop-thread-0 DefaultChannelPipeline : An exceptionCaught() event was fired, and it reached at the tail of the pipeline. It usually means the last handler in the pipeline did not handle the exception.
io.netty.handler.codec.DecoderException: javax.net.ssl.SSLException: Received fatal alert: certificate_unknown
	at io.netty.handler.codec.ByteToMessageDecoder.callDecode(ByteToMessageDecoder.java:459)
	at io.netty.handler.codec.ByteToMessageDecoder.channelRead(ByteToMessageDecoder.java:265)
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:362)
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:348)
	at io.netty.channel.AbstractChannelHandlerContext.fireChannelRead(AbstractChannelHandlerContext.java:340)
	at io.netty.channel.DefaultChannelPipeline$HeadContext.channelRead(DefaultChannelPipeline.java:1359)
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:362)
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:348)
	at io.netty.channel.DefaultChannelPipeline.fireChannelRead(DefaultChannelPipeline.java:935)
	at io.netty.channel.nio.AbstractNioByteChannel$NioByteUnsafe.read(AbstractNioByteChannel.java:134)
	at io.netty.channel.nio.NioEventLoop.processSelectedKey(NioEventLoop.java:645)
	at io.netty.channel.nio.NioEventLoop.processSelectedKeysOptimized(NioEventLoop.java:580)
	at io.netty.channel.nio.NioEventLoop.processSelectedKeys(NioEventLoop.java:497)
	at io.netty.channel.nio.NioEventLoop.run(NioEventLoop.java:459)
	at io.netty.util.concurrent.SingleThreadEventExecutor$5.run(SingleThreadEventExecutor.java:858)
	at java.lang.Thread.run(Unknown Source)
Caused by: javax.net.ssl.SSLException: Received fatal alert: certificate_unknown
	at sun.security.ssl.Alerts.getSSLException(Unknown Source)
	at sun.security.ssl.SSLEngineImpl.fatal(Unknown Source)
	at sun.security.ssl.SSLEngineImpl.fatal(Unknown Source)
	at sun.security.ssl.SSLEngineImpl.recvAlert(Unknown Source)
	at sun.security.ssl.SSLEngineImpl.readRecord(Unknown Source)
	at sun.security.ssl.SSLEngineImpl.readNetRecord(Unknown Source)
	at sun.security.ssl.SSLEngineImpl.unwrap(Unknown Source)
	at javax.net.ssl.SSLEngine.unwrap(Unknown Source)
	at io.netty.handler.ssl.SslHandler$SslEngineType$3.unwrap(SslHandler.java:281)
	at io.netty.handler.ssl.SslHandler.unwrap(SslHandler.java:1215)
	at io.netty.handler.ssl.SslHandler.decodeJdkCompatible(SslHandler.java:1127)
	at io.netty.handler.ssl.SslHandler.decode(SslHandler.java:1162)
	at io.netty.handler.codec.ByteToMessageDecoder.decodeRemovalReentryProtection(ByteToMessageDecoder.java:489)
	at io.netty.handler.codec.ByteToMessageDecoder.callDecode(ByteToMessageDecoder.java:428)
	... 15 common frames omitted
22:42:01.324 WARN  vert.x-eventloop-thread-0 DefaultChannelPipeline : An exceptionCaught() event was fired, and it reached at the tail of the pipeline. It usually means the last handler in the pipeline did not handle the exception.
io.netty.handler.codec.DecoderException: javax.net.ssl.SSLException: Received fatal alert: certificate_unknown
	at io.netty.handler.codec.ByteToMessageDecoder.callDecode(ByteToMessageDecoder.java:459)
	at io.netty.handler.codec.ByteToMessageDecoder.channelRead(ByteToMessageDecoder.java:265)
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:362)
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:348)
	at io.netty.channel.AbstractChannelHandlerContext.fireChannelRead(AbstractChannelHandlerContext.java:340)
	at io.netty.channel.DefaultChannelPipeline$HeadContext.channelRead(DefaultChannelPipeline.java:1359)
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:362)
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:348)
	at io.netty.channel.DefaultChannelPipeline.fireChannelRead(DefaultChannelPipeline.java:935)
	at io.netty.channel.nio.AbstractNioByteChannel$NioByteUnsafe.read(AbstractNioByteChannel.java:134)
	at io.netty.channel.nio.NioEventLoop.processSelectedKey(NioEventLoop.java:645)
	at io.netty.channel.nio.NioEventLoop.processSelectedKeysOptimized(NioEventLoop.java:580)
	at io.netty.channel.nio.NioEventLoop.processSelectedKeys(NioEventLoop.java:497)
	at io.netty.channel.nio.NioEventLoop.run(NioEventLoop.java:459)
	at io.netty.util.concurrent.SingleThreadEventExecutor$5.run(SingleThreadEventExecutor.java:858)
	at java.lang.Thread.run(Unknown Source)
Caused by: javax.net.ssl.SSLException: Received fatal alert: certificate_unknown
	at sun.security.ssl.Alerts.getSSLException(Unknown Source)
	at sun.security.ssl.SSLEngineImpl.fatal(Unknown Source)
	at sun.security.ssl.SSLEngineImpl.fatal(Unknown Source)
	at sun.security.ssl.SSLEngineImpl.recvAlert(Unknown Source)
	at sun.security.ssl.SSLEngineImpl.readRecord(Unknown Source)
	at sun.security.ssl.SSLEngineImpl.readNetRecord(Unknown Source)
	at sun.security.ssl.SSLEngineImpl.unwrap(Unknown Source)
	at javax.net.ssl.SSLEngine.unwrap(Unknown Source)
	at io.netty.handler.ssl.SslHandler$SslEngineType$3.unwrap(SslHandler.java:281)
	at io.netty.handler.ssl.SslHandler.unwrap(SslHandler.java:1215)
	at io.netty.handler.ssl.SslHandler.decodeJdkCompatible(SslHandler.java:1127)
	at io.netty.handler.ssl.SslHandler.decode(SslHandler.java:1162)
	at io.netty.handler.codec.ByteToMessageDecoder.decodeRemovalReentryProtection(ByteToMessageDecoder.java:489)
	at io.netty.handler.codec.ByteToMessageDecoder.callDecode(ByteToMessageDecoder.java:428)
	... 15 common frames omitted
22:42:01.325 WARN  vert.x-eventloop-thread-0 DefaultChannelPipeline : An exceptionCaught() event was fired, and it reached at the tail of the pipeline. It usually means the last handler in the pipeline did not handle the exception.
io.netty.handler.codec.DecoderException: javax.net.ssl.SSLException: Received fatal alert: certificate_unknown
	at io.netty.handler.codec.ByteToMessageDecoder.callDecode(ByteToMessageDecoder.java:459)
	at io.netty.handler.codec.ByteToMessageDecoder.channelRead(ByteToMessageDecoder.java:265)
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:362)
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:348)
	at io.netty.channel.AbstractChannelHandlerContext.fireChannelRead(AbstractChannelHandlerContext.java:340)
	at io.netty.channel.DefaultChannelPipeline$HeadContext.channelRead(DefaultChannelPipeline.java:1359)
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:362)
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:348)
	at io.netty.channel.DefaultChannelPipeline.fireChannelRead(DefaultChannelPipeline.java:935)
	at io.netty.channel.nio.AbstractNioByteChannel$NioByteUnsafe.read(AbstractNioByteChannel.java:134)
	at io.netty.channel.nio.NioEventLoop.processSelectedKey(NioEventLoop.java:645)
	at io.netty.channel.nio.NioEventLoop.processSelectedKeysOptimized(NioEventLoop.java:580)
	at io.netty.channel.nio.NioEventLoop.processSelectedKeys(NioEventLoop.java:497)
	at io.netty.channel.nio.NioEventLoop.run(NioEventLoop.java:459)
	at io.netty.util.concurrent.SingleThreadEventExecutor$5.run(SingleThreadEventExecutor.java:858)
	at java.lang.Thread.run(Unknown Source)
Caused by: javax.net.ssl.SSLException: Received fatal alert: certificate_unknown
	at sun.security.ssl.Alerts.getSSLException(Unknown Source)
	at sun.security.ssl.SSLEngineImpl.fatal(Unknown Source)
	at sun.security.ssl.SSLEngineImpl.fatal(Unknown Source)
	at sun.security.ssl.SSLEngineImpl.recvAlert(Unknown Source)
	at sun.security.ssl.SSLEngineImpl.readRecord(Unknown Source)
	at sun.security.ssl.SSLEngineImpl.readNetRecord(Unknown Source)
	at sun.security.ssl.SSLEngineImpl.unwrap(Unknown Source)
	at javax.net.ssl.SSLEngine.unwrap(Unknown Source)
	at io.netty.handler.ssl.SslHandler$SslEngineType$3.unwrap(SslHandler.java:281)
	at io.netty.handler.ssl.SslHandler.unwrap(SslHandler.java:1215)
	at io.netty.handler.ssl.SslHandler.decodeJdkCompatible(SslHandler.java:1127)
	at io.netty.handler.ssl.SslHandler.decode(SslHandler.java:1162)
	at io.netty.handler.codec.ByteToMessageDecoder.decodeRemovalReentryProtection(ByteToMessageDecoder.java:489)
	at io.netty.handler.codec.ByteToMessageDecoder.callDecode(ByteToMessageDecoder.java:428)
	... 15 common frames omitted
22:42:01.470 INFO  vert.x-eventloop-thread-0 BaseServiceProxy     : -> GET /sso/Templates/otp-responsive.css?1622551322838,200|469ms
22:42:01.613 INFO  vert.x-eventloop-thread-0 BaseServiceProxy     : -> GET /scripts/common/js/switchery/switchery-init.js,200|141ms
22:42:01.618 INFO  vert.x-eventloop-thread-0 BaseServiceProxy     : -> GET /scripts/common/js/icheck-1.0.2/icheck.min.js,200|144ms
22:42:01.627 WARN  vert.x-eventloop-thread-0 DefaultChannelPipeline : An exceptionCaught() event was fired, and it reached at the tail of the pipeline. It usually means the last handler in the pipeline did not handle the exception.
io.netty.handler.codec.DecoderException: javax.net.ssl.SSLException: Received fatal alert: certificate_unknown
	at io.netty.handler.codec.ByteToMessageDecoder.callDecode(ByteToMessageDecoder.java:459)
	at io.netty.handler.codec.ByteToMessageDecoder.channelRead(ByteToMessageDecoder.java:265)
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:362)
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:348)
	at io.netty.channel.AbstractChannelHandlerContext.fireChannelRead(AbstractChannelHandlerContext.java:340)
	at io.netty.channel.DefaultChannelPipeline$HeadContext.channelRead(DefaultChannelPipeline.java:1359)
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:362)
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:348)
	at io.netty.channel.DefaultChannelPipeline.fireChannelRead(DefaultChannelPipeline.java:935)
	at io.netty.channel.nio.AbstractNioByteChannel$NioByteUnsafe.read(AbstractNioByteChannel.java:134)
	at io.netty.channel.nio.NioEventLoop.processSelectedKey(NioEventLoop.java:645)
	at io.netty.channel.nio.NioEventLoop.processSelectedKeysOptimized(NioEventLoop.java:580)
	at io.netty.channel.nio.NioEventLoop.processSelectedKeys(NioEventLoop.java:497)
	at io.netty.channel.nio.NioEventLoop.run(NioEventLoop.java:459)
	at io.netty.util.concurrent.SingleThreadEventExecutor$5.run(SingleThreadEventExecutor.java:858)
	at java.lang.Thread.run(Unknown Source)
Caused by: javax.net.ssl.SSLException: Received fatal alert: certificate_unknown
	at sun.security.ssl.Alerts.getSSLException(Unknown Source)
	at sun.security.ssl.SSLEngineImpl.fatal(Unknown Source)
	at sun.security.ssl.SSLEngineImpl.fatal(Unknown Source)
	at sun.security.ssl.SSLEngineImpl.recvAlert(Unknown Source)
	at sun.security.ssl.SSLEngineImpl.readRecord(Unknown Source)
	at sun.security.ssl.SSLEngineImpl.readNetRecord(Unknown Source)
	at sun.security.ssl.SSLEngineImpl.unwrap(Unknown Source)
	at javax.net.ssl.SSLEngine.unwrap(Unknown Source)
	at io.netty.handler.ssl.SslHandler$SslEngineType$3.unwrap(SslHandler.java:281)
	at io.netty.handler.ssl.SslHandler.unwrap(SslHandler.java:1215)
	at io.netty.handler.ssl.SslHandler.decodeJdkCompatible(SslHandler.java:1127)
	at io.netty.handler.ssl.SslHandler.decode(SslHandler.java:1162)
	at io.netty.handler.codec.ByteToMessageDecoder.decodeRemovalReentryProtection(ByteToMessageDecoder.java:489)
	at io.netty.handler.codec.ByteToMessageDecoder.callDecode(ByteToMessageDecoder.java:428)
	... 15 common frames omitted
22:42:01.628 WARN  vert.x-eventloop-thread-0 DefaultChannelPipeline : An exceptionCaught() event was fired, and it reached at the tail of the pipeline. It usually means the last handler in the pipeline did not handle the exception.
io.netty.handler.codec.DecoderException: javax.net.ssl.SSLException: Received fatal alert: certificate_unknown
	at io.netty.handler.codec.ByteToMessageDecoder.callDecode(ByteToMessageDecoder.java:459)
	at io.netty.handler.codec.ByteToMessageDecoder.channelRead(ByteToMessageDecoder.java:265)
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:362)
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:348)
	at io.netty.channel.AbstractChannelHandlerContext.fireChannelRead(AbstractChannelHandlerContext.java:340)
	at io.netty.channel.DefaultChannelPipeline$HeadContext.channelRead(DefaultChannelPipeline.java:1359)
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:362)
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:348)
	at io.netty.channel.DefaultChannelPipeline.fireChannelRead(DefaultChannelPipeline.java:935)
	at io.netty.channel.nio.AbstractNioByteChannel$NioByteUnsafe.read(AbstractNioByteChannel.java:134)
	at io.netty.channel.nio.NioEventLoop.processSelectedKey(NioEventLoop.java:645)
	at io.netty.channel.nio.NioEventLoop.processSelectedKeysOptimized(NioEventLoop.java:580)
	at io.netty.channel.nio.NioEventLoop.processSelectedKeys(NioEventLoop.java:497)
	at io.netty.channel.nio.NioEventLoop.run(NioEventLoop.java:459)
	at io.netty.util.concurrent.SingleThreadEventExecutor$5.run(SingleThreadEventExecutor.java:858)
	at java.lang.Thread.run(Unknown Source)
Caused by: javax.net.ssl.SSLException: Received fatal alert: certificate_unknown
	at sun.security.ssl.Alerts.getSSLException(Unknown Source)
	at sun.security.ssl.SSLEngineImpl.fatal(Unknown Source)
	at sun.security.ssl.SSLEngineImpl.fatal(Unknown Source)
	at sun.security.ssl.SSLEngineImpl.recvAlert(Unknown Source)
	at sun.security.ssl.SSLEngineImpl.readRecord(Unknown Source)
	at sun.security.ssl.SSLEngineImpl.readNetRecord(Unknown Source)
	at sun.security.ssl.SSLEngineImpl.unwrap(Unknown Source)
	at javax.net.ssl.SSLEngine.unwrap(Unknown Source)
	at io.netty.handler.ssl.SslHandler$SslEngineType$3.unwrap(SslHandler.java:281)
	at io.netty.handler.ssl.SslHandler.unwrap(SslHandler.java:1215)
	at io.netty.handler.ssl.SslHandler.decodeJdkCompatible(SslHandler.java:1127)
	at io.netty.handler.ssl.SslHandler.decode(SslHandler.java:1162)
	at io.netty.handler.codec.ByteToMessageDecoder.decodeRemovalReentryProtection(ByteToMessageDecoder.java:489)
	at io.netty.handler.codec.ByteToMessageDecoder.callDecode(ByteToMessageDecoder.java:428)
	... 15 common frames omitted
22:42:01.631 WARN  vert.x-eventloop-thread-0 DefaultChannelPipeline : An exceptionCaught() event was fired, and it reached at the tail of the pipeline. It usually means the last handler in the pipeline did not handle the exception.
io.netty.handler.codec.DecoderException: javax.net.ssl.SSLException: Received fatal alert: certificate_unknown
	at io.netty.handler.codec.ByteToMessageDecoder.callDecode(ByteToMessageDecoder.java:459)
	at io.netty.handler.codec.ByteToMessageDecoder.channelRead(ByteToMessageDecoder.java:265)
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:362)
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:348)
	at io.netty.channel.AbstractChannelHandlerContext.fireChannelRead(AbstractChannelHandlerContext.java:340)
	at io.netty.channel.DefaultChannelPipeline$HeadContext.channelRead(DefaultChannelPipeline.java:1359)
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:362)
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:348)
	at io.netty.channel.DefaultChannelPipeline.fireChannelRead(DefaultChannelPipeline.java:935)
	at io.netty.channel.nio.AbstractNioByteChannel$NioByteUnsafe.read(AbstractNioByteChannel.java:134)
	at io.netty.channel.nio.NioEventLoop.processSelectedKey(NioEventLoop.java:645)
	at io.netty.channel.nio.NioEventLoop.processSelectedKeysOptimized(NioEventLoop.java:580)
	at io.netty.channel.nio.NioEventLoop.processSelectedKeys(NioEventLoop.java:497)
	at io.netty.channel.nio.NioEventLoop.run(NioEventLoop.java:459)
	at io.netty.util.concurrent.SingleThreadEventExecutor$5.run(SingleThreadEventExecutor.java:858)
	at java.lang.Thread.run(Unknown Source)
Caused by: javax.net.ssl.SSLException: Received fatal alert: certificate_unknown
	at sun.security.ssl.Alerts.getSSLException(Unknown Source)
	at sun.security.ssl.SSLEngineImpl.fatal(Unknown Source)
	at sun.security.ssl.SSLEngineImpl.fatal(Unknown Source)
	at sun.security.ssl.SSLEngineImpl.recvAlert(Unknown Source)
	at sun.security.ssl.SSLEngineImpl.readRecord(Unknown Source)
	at sun.security.ssl.SSLEngineImpl.readNetRecord(Unknown Source)
	at sun.security.ssl.SSLEngineImpl.unwrap(Unknown Source)
	at javax.net.ssl.SSLEngine.unwrap(Unknown Source)
	at io.netty.handler.ssl.SslHandler$SslEngineType$3.unwrap(SslHandler.java:281)
	at io.netty.handler.ssl.SslHandler.unwrap(SslHandler.java:1215)
	at io.netty.handler.ssl.SslHandler.decodeJdkCompatible(SslHandler.java:1127)
	at io.netty.handler.ssl.SslHandler.decode(SslHandler.java:1162)
	at io.netty.handler.codec.ByteToMessageDecoder.decodeRemovalReentryProtection(ByteToMessageDecoder.java:489)
	at io.netty.handler.codec.ByteToMessageDecoder.callDecode(ByteToMessageDecoder.java:428)
	... 15 common frames omitted
22:42:01.632 WARN  vert.x-eventloop-thread-0 DefaultChannelPipeline : An exceptionCaught() event was fired, and it reached at the tail of the pipeline. It usually means the last handler in the pipeline did not handle the exception.
io.netty.handler.codec.DecoderException: javax.net.ssl.SSLException: Received fatal alert: certificate_unknown
	at io.netty.handler.codec.ByteToMessageDecoder.callDecode(ByteToMessageDecoder.java:459)
	at io.netty.handler.codec.ByteToMessageDecoder.channelRead(ByteToMessageDecoder.java:265)
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:362)
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:348)
	at io.netty.channel.AbstractChannelHandlerContext.fireChannelRead(AbstractChannelHandlerContext.java:340)
	at io.netty.channel.DefaultChannelPipeline$HeadContext.channelRead(DefaultChannelPipeline.java:1359)
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:362)
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:348)
	at io.netty.channel.DefaultChannelPipeline.fireChannelRead(DefaultChannelPipeline.java:935)
	at io.netty.channel.nio.AbstractNioByteChannel$NioByteUnsafe.read(AbstractNioByteChannel.java:134)
	at io.netty.channel.nio.NioEventLoop.processSelectedKey(NioEventLoop.java:645)
	at io.netty.channel.nio.NioEventLoop.processSelectedKeysOptimized(NioEventLoop.java:580)
	at io.netty.channel.nio.NioEventLoop.processSelectedKeys(NioEventLoop.java:497)
	at io.netty.channel.nio.NioEventLoop.run(NioEventLoop.java:459)
	at io.netty.util.concurrent.SingleThreadEventExecutor$5.run(SingleThreadEventExecutor.java:858)
	at java.lang.Thread.run(Unknown Source)
Caused by: javax.net.ssl.SSLException: Received fatal alert: certificate_unknown
	at sun.security.ssl.Alerts.getSSLException(Unknown Source)
	at sun.security.ssl.SSLEngineImpl.fatal(Unknown Source)
	at sun.security.ssl.SSLEngineImpl.fatal(Unknown Source)
	at sun.security.ssl.SSLEngineImpl.recvAlert(Unknown Source)
	at sun.security.ssl.SSLEngineImpl.readRecord(Unknown Source)
	at sun.security.ssl.SSLEngineImpl.readNetRecord(Unknown Source)
	at sun.security.ssl.SSLEngineImpl.unwrap(Unknown Source)
	at javax.net.ssl.SSLEngine.unwrap(Unknown Source)
	at io.netty.handler.ssl.SslHandler$SslEngineType$3.unwrap(SslHandler.java:281)
	at io.netty.handler.ssl.SslHandler.unwrap(SslHandler.java:1215)
	at io.netty.handler.ssl.SslHandler.decodeJdkCompatible(SslHandler.java:1127)
	at io.netty.handler.ssl.SslHandler.decode(SslHandler.java:1162)
	at io.netty.handler.codec.ByteToMessageDecoder.decodeRemovalReentryProtection(ByteToMessageDecoder.java:489)
	at io.netty.handler.codec.ByteToMessageDecoder.callDecode(ByteToMessageDecoder.java:428)
	... 15 common frames omitted
22:42:01.765 INFO  vert.x-eventloop-thread-0 BaseServiceProxy     : -> GET /scripts/common/js/icheck-1.0.2/icheck-init.js,200|148ms
22:42:01.919 INFO  vert.x-eventloop-thread-0 BaseServiceProxy     : -> GET /scripts/common/js/jquery-ui-1.12.1/jquery-ui.min.js,200|151ms
22:42:02.087 INFO  vert.x-eventloop-thread-0 BaseServiceProxy     : -> GET /scripts/common/js/switchery/switchery.min.js,200|144ms
22:42:02.215 INFO  vert.x-eventloop-thread-0 BaseServiceProxy     : -> GET /sso/CombineFiles,200|592ms
22:42:02.937 INFO  vert.x-eventloop-thread-0 BaseServiceProxy     : -> GET /scripts/common/js/bootstrap-4.3.1/bootstrap.bundle.min.js,200|1000ms
22:42:03.108 INFO  vert.x-eventloop-thread-0 BaseServiceProxy     : -> GET /scripts/common/js/bootstrap-switch-3.3.2/bootstrap-switch.min.js,200|1169ms
22:42:03.506 INFO  vert.x-eventloop-thread-0 BaseServiceProxy     : -> GET /images/2015/btn-app-apple.png,200|145ms
22:42:03.653 INFO  vert.x-eventloop-thread-0 BaseServiceProxy     : -> GET /images/2015/btn-app-google.png,200|141ms
22:42:03.685 INFO  vert.x-eventloop-thread-0 BaseServiceProxy     : -> GET /images/2015/btn-app-tws.png,200|141ms
22:42:03.695 INFO  vert.x-eventloop-thread-0 BaseServiceProxy     : -> GET /images/2015/button_discover_apis.png,200|145ms
22:42:04.233 INFO  vert.x-eventloop-thread-0 BaseServiceProxy     : -> GET /images/web/favicons/home-screen-icon-128x128.png,200|141ms
22:42:07.864 INFO  vert.x-eventloop-thread-0 BaseServiceProxy     : -> GET /sso/Authenticator;jsessionid=CDEE94B76061601ECE67DA5E77B5BBD8.ny5wwwdam1-internet?38014,200|645ms
22:42:10.875 INFO  vert.x-eventloop-thread-0 BaseServiceProxy     : -> GET /sso/Authenticator;jsessionid=CDEE94B76061601ECE67DA5E77B5BBD8.ny5wwwdam1-internet?97854,200|1370ms
22:42:10.875 INFO  vert.x-eventloop-thread-0 CookieManager        : set cookies [web=802397474; Domain=.ibkr.com; Expires=Wed, 01-Jun-2022 12:42:12 GMT; Path=/; Secure;Secure;SameSite=None]
22:42:10.876 DEBUG vert.x-eventloop-thread-0 BaseServiceProxy     : Remapping Set-cookies [web=802397474; Domain=.ibkr.com; Expires=Wed, 01-Jun-2022 12:42:12 GMT; Path=/; Secure;Secure;SameSite=None] -> 
22:42:11.801 INFO  vert.x-eventloop-thread-0 BaseServiceProxy     : -> GET /sso/Dispatcher;jsessionid=CDEE94B76061601ECE67DA5E77B5BBD8.ny5wwwdam1-internet,302|908ms
22:42:11.801 INFO  vert.x-eventloop-thread-0 CookieManager        : set cookies [USERID=59143911; Domain=.ibkr.com; Path=/;Secure;HttpOnly;SameSite=None, REGION=us; Domain=.ibkr.com; Path=/;Secure;HttpOnly;SameSite=None, USERID=59143911; Domain=.interactiveadvisors.com; Path=/; Secure;Secure;HttpOnly;SameSite=None, REGION=us; Domain=.interactiveadvisors.com; Path=/; Secure;Secure;HttpOnly;SameSite=None, XYZAB_AM.LOGIN=07f71c0e37624bdc485e23234589778a439db8b0; Domain=.interactiveadvisors.com; Path=/; Secure;Secure;SameSite=None, XYZAB=07f71c0e37624bdc485e23234589778a439db8b0; Domain=.interactiveadvisors.com; Path=/; Secure;Secure;SameSite=None]
22:42:11.801 DEBUG vert.x-eventloop-thread-0 BaseServiceProxy     : Remapping Set-cookies [USERID=59143911; Domain=.ibkr.com; Path=/;Secure;HttpOnly;SameSite=None, REGION=us; Domain=.ibkr.com; Path=/;Secure;HttpOnly;SameSite=None, USERID=59143911; Domain=.interactiveadvisors.com; Path=/; Secure;Secure;HttpOnly;SameSite=None, REGION=us; Domain=.interactiveadvisors.com; Path=/; Secure;Secure;HttpOnly;SameSite=None, XYZAB_AM.LOGIN=07f71c0e37624bdc485e23234589778a439db8b0; Domain=.interactiveadvisors.com; Path=/; Secure;Secure;SameSite=None, XYZAB=07f71c0e37624bdc485e23234589778a439db8b0; Domain=.interactiveadvisors.com; Path=/; Secure;Secure;SameSite=None] -> 
22:42:11.802 INFO  vert.x-eventloop-thread-0 GatewayHttpProxy     : Client login succeeds
22:42:11.804 INFO  vert.x-eventloop-thread-0 AmService            : authenticating with AM https://api.ibkr.com/AccountManagement/OneBarAuthentication?json=1
22:42:12.221 INFO  vert.x-eventloop-thread-0 BaseServiceProxy     : -> GET /v1/api/sso/validate?gw=1,500|419ms
22:42:12.222 WARN  vert.x-eventloop-thread-0 BaseServiceProxy     : failed /v1/api/sso/validate?gw=1 | reason <!DOCTYPE HTML PUBLIC "-//IETF//DTD HTML 2.0//EN">
<html><head>
<title>500 Internal Server Error</title>
</head><body>
<h1>Internal Server Error</h1>
<p>The server encountered an internal error or
misconfiguration and was unable to complete
your request.</p>
<p>Please contact the server administrator at 
 webmaster@interactivebrokers.com to inform them of the time this error occurred,
 and the actions you performed just before this error.</p>
<p>More information about this error may be available
in the server error log.</p>
<p>Additionally, a 500 Internal Server Error
error was encountered while trying to use an ErrorDocument to handle the request.</p>
</body></html>

22:42:12.223 DEBUG vert.x-eventloop-thread-0 ClientPortalService  : portal validate response: null
22:42:12.223 ERROR vert.x-eventloop-thread-0 GatewayHttpProxy     : authentication to cp failed, retry 1
io.vertx.core.impl.NoStackTraceThrowable: <!DOCTYPE HTML PUBLIC "-//IETF//DTD HTML 2.0//EN">
<html><head>
<title>500 Internal Server Error</title>
</head><body>
<h1>Internal Server Error</h1>
<p>The server encountered an internal error or
misconfiguration and was unable to complete
your request.</p>
<p>Please contact the server administrator at 
 webmaster@interactivebrokers.com to inform them of the time this error occurred,
 and the actions you performed just before this error.</p>
<p>More information about this error may be available
in the server error log.</p>
<p>Additionally, a 500 Internal Server Error
error was encountered while trying to use an ErrorDocument to handle the request.</p>
</body></html>

22:42:12.224 WARN  vert.x-eventloop-thread-0 BaseServiceProxy     : failed /v1/api/sso/validate?gw=1 | reason 500
22:42:12.224 DEBUG vert.x-eventloop-thread-0 ClientPortalService  : portal validate response: null
22:42:12.224 ERROR vert.x-eventloop-thread-0 GatewayHttpProxy     : authentication to cp failed, retry 1
io.vertx.core.impl.NoStackTraceThrowable: 500
22:42:12.225 DEBUG vert.x-eventloop-thread-0 BaseServiceProxy     : forward event CP_LOGIN_FAILED, payload none
22:42:12.225 INFO  vert.x-eventloop-thread-0 GatewayHttpProxy     : Unhandled event CP_LOGIN_FAILED
22:42:12.225 DEBUG vert.x-eventloop-thread-0 BaseServiceProxy     : forward event CP_LOGIN_FAILED, payload none
22:42:12.225 INFO  vert.x-eventloop-thread-0 GatewayHttpProxy     : Unhandled event CP_LOGIN_FAILED
22:42:12.230 INFO  vert.x-eventloop-thread-0 BaseServiceProxy     : -> GET /AccountManagement/OneBarAuthentication?json=1,404|425ms
22:42:12.230 WARN  vert.x-eventloop-thread-0 BaseServiceProxy     : failed /AccountManagement/OneBarAuthentication?json=1 | reason <!DOCTYPE HTML PUBLIC "-//IETF//DTD HTML 2.0//EN">
<html><head>
<title>404 Not Found</title>
</head><body>
<h1>Not Found</h1>
<p>The requested URL was not found on this server.</p>
<p>Additionally, a 404 Not Found
error was encountered while trying to use an ErrorDocument to handle the request.</p>
</body></html>

22:42:12.230 ERROR vert.x-eventloop-thread-0 AmService            : am authentication failed
io.vertx.core.impl.NoStackTraceThrowable: <!DOCTYPE HTML PUBLIC "-//IETF//DTD HTML 2.0//EN">
<html><head>
<title>404 Not Found</title>
</head><body>
<h1>Not Found</h1>
<p>The requested URL was not found on this server.</p>
<p>Additionally, a 404 Not Found
error was encountered while trying to use an ErrorDocument to handle the request.</p>
</body></html>

22:42:12.230 WARN  vert.x-eventloop-thread-0 BaseServiceProxy     : failed /AccountManagement/OneBarAuthentication?json=1 | reason 404
22:42:12.230 ERROR vert.x-eventloop-thread-0 AmService            : am authentication failed
io.vertx.core.impl.NoStackTraceThrowable: 404
22:42:16.294 INFO  vert.x-eventloop-thread-0 BaseServiceProxy     : -> GET /v1/api/sso/validate?gw=1,200|1070ms
22:42:16.294 DEBUG vert.x-eventloop-thread-0 ClientPortalService  : portal validate response: {"LOGIN_TYPE":1,"LANDING_APP":"UNIVERSAL","IS_PENDING_APPLICANT":false,"SF_ENABLED":false,"AUTH_TIME":1622551332917,"CREDENTIAL":"tymraf123","SF_CONFIG":"","USER_NAME":"tymraf123","CREDENTIAL_TYPE":0,"RESULT":true,"IS_FREE_TRIAL":false,"IP":"120.156.178.227","USER_ID":59143911,"EXPIRES":1622551338783,"TOKEN":"07f71c0e37624bdc485e23234589778a439db8b0","took":916,"features":{"env":"PROD","wlms":true,"realtime":true,"bond":true,"optionChains":true,"calendar":true,"newMf":true}}
22:42:16.432 INFO  vert.x-eventloop-thread-0 GatewayHttpProxy     : authenticated to cp
22:42:16.433 DEBUG vert.x-eventloop-thread-0 BaseServiceProxy     : cancel timer SsoService 0
22:42:16.433 DEBUG vert.x-eventloop-thread-0 BaseServiceProxy     : start timer 300000 SsoService
22:42:16.435 DEBUG vert.x-eventloop-thread-0 BaseServiceProxy     : forward event ON_SSO_VALIDATION, payload 59143911
22:42:16.435 INFO  vert.x-eventloop-thread-0 BaseServiceProxy     : -> GET /v1/api/sso/validate?gw=1,200|1211ms
22:42:16.435 DEBUG vert.x-eventloop-thread-0 ClientPortalService  : portal validate response: {"LOGIN_TYPE":1,"LANDING_APP":"UNIVERSAL","IS_PENDING_APPLICANT":false,"SF_ENABLED":false,"AUTH_TIME":1622551332917,"CREDENTIAL":"tymraf123","SF_CONFIG":"","USER_NAME":"tymraf123","CREDENTIAL_TYPE":0,"RESULT":true,"IS_FREE_TRIAL":false,"IP":"120.156.178.227","USER_ID":59143911,"EXPIRES":1622551338783,"TOKEN":"07f71c0e37624bdc485e23234589778a439db8b0","took":916,"features":{"env":"PROD","wlms":true,"realtime":true,"bond":true,"optionChains":true,"calendar":true,"newMf":true},"isGw":true}
22:42:16.436 INFO  vert.x-eventloop-thread-0 GatewayHttpProxy     : authenticated to cp
22:42:16.436 DEBUG vert.x-eventloop-thread-0 BaseServiceProxy     : cancel timer SsoService 2
22:42:16.436 DEBUG vert.x-eventloop-thread-0 BaseServiceProxy     : start timer 300000 SsoService
22:42:16.437 DEBUG vert.x-eventloop-thread-0 BaseServiceProxy     : forward event ON_SSO_VALIDATION, payload 59143911
22:42:16.584 INFO  vert.x-eventloop-thread-0 BaseServiceProxy     : -> GET /v1/api/one/user,200|148ms
22:42:16.585 INFO  vert.x-eventloop-thread-0 ClientPortalService  : get user reply {"ispaper":false,"islite":false,"has2fa":false,"username":"tymraf123","features":{"env":"PROD","wlms":true,"realtime":true,"bond":true,"optionChains":true,"calendar":true,"newMf":true},"applicants":[{"id":7808734,"type":"ORG","entityId":163825041,"businessType":"INSTITUTIONAL","legalCountry":{"name":"Australia","alpha3":"AUS"},"nlcode":"en_US"}],"uar":{"portfolioAnalyst":true,"userInfo":false,"messageCenter":true,"accountDetails":true,"tradingRestrictions":true,"tws":true,"fyi":true,"voting":true,"forum":true,"recentTransactions":true},"accts":{"U5499758":{"clearingStatus":"O","openDate":1613318400000,"isFunded":false,"tradingPermissions":["WAR","STK","CASH"],"isFAClient":false}},"props":{"readOnlySession":null,"isIBAMClient":false},"firstApprovedDate":1619107200000}
22:42:16.853 INFO  vert.x-eventloop-thread-0 BaseServiceProxy     : -> GET /v1/api/one/user,200|421ms
22:42:16.853 INFO  vert.x-eventloop-thread-0 ClientPortalService  : get user reply {"ispaper":false,"islite":false,"has2fa":false,"username":"tymraf123","features":{"env":"PROD","wlms":true,"realtime":true,"bond":true,"optionChains":true,"calendar":true,"newMf":true},"applicants":[{"id":7808734,"type":"ORG","entityId":163825041,"businessType":"INSTITUTIONAL","legalCountry":{"name":"Australia","alpha3":"AUS"},"nlcode":"en_US"}],"uar":{"portfolioAnalyst":true,"userInfo":false,"messageCenter":true,"accountDetails":true,"tradingRestrictions":true,"tws":true,"fyi":true,"voting":true,"forum":true,"recentTransactions":true},"accts":{"U5499758":{"clearingStatus":"O","openDate":1613318400000,"isFunded":false,"tradingPermissions":["WAR","STK","CASH"],"isFAClient":false}},"props":{"readOnlySession":null,"isIBAMClient":false},"firstApprovedDate":1619107200000}
22:42:17.335 INFO  vert.x-eventloop-thread-0 SsoService           : sso init status=200, url=https://api.ibkr.com/sso/AuthenticateSSODH?ACTION=INIT
22:42:17.337 DEBUG vert.x-eventloop-thread-0 SsoService           : sso init reponse: <ib_auth_res>
<dh_params>
<hash>SHA-1</hash>
<p>97b5fbcc7d462ce90e4e8cacb74f97bcfe42b2a9376e9cbe05ba03c93c26f3dac061e3b22c035f2215550b6b5833a0ac779173d8e5765db04526306bfdb5ad5b</p>
<g>ada72a1c424119fdb1fecc8ad17307e8f4e36fa08df8f3eb2ffd4f799a9259154be47c1f1b72d951d22f143dc1d9bb84cc26e170bdae05014df1f5da9b2f2c2b</g>
<A>3d2fdca96c6a02acdb854a0bc69f9f179493f0fbd9d6f1373118bcc1a1455b526fa6aef8c3383a331fd9ae8ee2d62191913b1e2fdbb1bf117c3275a03848633b</A>
<challenge>f50fb10f97ff475c63c58f0582d99234f6d22d0</challenge>
<info></info>
</dh_params>
</ib_auth_res>

22:42:17.337 DEBUG vert.x-eventloop-thread-0 SsoService           : p 97b5fbcc7d462ce90e4e8cacb74f97bcfe42b2a9376e9cbe05ba03c93c26f3dac061e3b22c035f2215550b6b5833a0ac779173d8e5765db04526306bfdb5ad5b
22:42:17.338 DEBUG vert.x-eventloop-thread-0 SsoService           : K f26bddb2f9dcc1fc4362c0374d4810cc7869949b
22:42:17.338 DEBUG vert.x-eventloop-thread-0 SsoService           : chlRes 200511c99d06322f6baea3be33645109130ea480
22:42:17.339 DEBUG vert.x-eventloop-thread-0 SsoService           : B 46b51f5d395eb00365822e534b1daa4fd2d6dc84532b41a3be4f0f767800020597b237d7b6878faec85d326ada24f75fc89c1d1821774f39b9595bf069e3883
22:42:17.397 INFO  vert.x-eventloop-thread-0 SsoService           : sso init status=200, url=https://api.ibkr.com/sso/AuthenticateSSODH?ACTION=INIT
22:42:17.398 DEBUG vert.x-eventloop-thread-0 SsoService           : sso init reponse: <ib_auth_res>
<dh_params>
<hash>SHA-1</hash>
<p>8dbf4da11ef89d5dda4cfc2150cb54e229b1f065dfe40af90256790d2be40b8dea921b0786af06d217866c5b08e83cb7632646c61d04ba3201895a98dfbc385b</p>
<g>a31a83d61ce7eca8b74bcfa6d5cd6c77c20171d4b81bbd07abae39a3092e4c2e21329daaf33331ed8a6cfa5b859fd6dd895604d9bc305a9fa2ff03b90c864ad5</g>
<A>566190460b335f0c1a96a7d410a34e51189c6de985a7a535701971ce3b74654f0859eb7490d421f17e540b6a2d8dcc27d7c65ed308c45926ba5191c90cddc884</A>
<challenge>9484564a62ac715c16da416f0389719ea0fa034a</challenge>
<info></info>
</dh_params>
</ib_auth_res>

22:42:17.398 DEBUG vert.x-eventloop-thread-0 SsoService           : p 8dbf4da11ef89d5dda4cfc2150cb54e229b1f065dfe40af90256790d2be40b8dea921b0786af06d217866c5b08e83cb7632646c61d04ba3201895a98dfbc385b
22:42:17.398 DEBUG vert.x-eventloop-thread-0 SsoService           : K 529d1942780228c4c9bbb8c501f196f2a97ec602
22:42:17.398 DEBUG vert.x-eventloop-thread-0 SsoService           : chlRes 759736950e3f12e65aa9862626268a41ca37eb93
22:42:17.399 DEBUG vert.x-eventloop-thread-0 SsoService           : B 23f2020325f41340ca414faf182fad35c1f7ae9ed9160183a1fda41f91a9349e54c93facfb0624914b1539b131dd1931a3fdef42f8b6faa12ebcc3438d0de6cb
22:42:17.740 INFO  vert.x-eventloop-thread-0 SsoService           : sso complete url status=200, url=https://api.ibkr.com/sso/AuthenticateSSODH?ACTION=COMPLETE&B=46b51f5d395eb00365822e534b1daa4fd2d6dc84532b41a3be4f0f767800020597b237d7b6878faec85d326ada24f75fc89c1d1821774f39b9595bf069e3883&CHRESPONSE=200511c99d06322f6baea3be33645109130ea480
22:42:17.742 DEBUG vert.x-eventloop-thread-0 SsoService           : challenge reply:
<ib_auth_res>
<error>
</error>
</ib_auth_res>

22:42:17.742 DEBUG vert.x-eventloop-thread-0 BaseServiceProxy     : forward event ON_SSODH_COMPLETED, payload none
22:42:17.742 DEBUG vert.x-eventloop-thread-0 GatewayHttpProxy     : delaying authentication for 3000
22:42:17.821 INFO  vert.x-eventloop-thread-0 TstTokenUtils        : Current MAC address : 00-15-5D-91-33-2C
22:42:17.821 DEBUG vert.x-eventloop-thread-0 SsoService           : tst token generated: 10a01310119bc317166808adf3de918c4e643d24
22:42:18.021 INFO  vert.x-eventloop-thread-0 SsoService           : sso complete url status=200, url=https://api.ibkr.com/sso/AuthenticateSSODH?ACTION=COMPLETE&B=23f2020325f41340ca414faf182fad35c1f7ae9ed9160183a1fda41f91a9349e54c93facfb0624914b1539b131dd1931a3fdef42f8b6faa12ebcc3438d0de6cb&CHRESPONSE=759736950e3f12e65aa9862626268a41ca37eb93
22:42:18.021 DEBUG vert.x-eventloop-thread-0 SsoService           : challenge reply:
<ib_auth_res>
<error>
</error>
</ib_auth_res>

22:42:18.021 DEBUG vert.x-eventloop-thread-0 BaseServiceProxy     : forward event ON_SSODH_COMPLETED, payload none
22:42:18.021 DEBUG vert.x-eventloop-thread-0 GatewayHttpProxy     : delaying authentication for 3000
22:42:18.409 INFO  vert.x-eventloop-thread-0 SsoService           : publish tst token status=200, url=https://api.ibkr.com/sso/Authenticator?ACTION=PUBLISH_TST&RESP_TYPE=JSON&DEVICE_ID=10fa9a34|00-15-5D-91-33-2C
22:42:18.409 DEBUG vert.x-eventloop-thread-0 SsoService           : publish tst token reply:
{"ST_EXPIRY_TIME":108000,"ST_EXPIRY_END":1622659340792,"USER_ID":59143911,"RESULT":true}
22:42:20.196 WARN  vert.x-eventloop-thread-0 DefaultChannelPipeline : An exceptionCaught() event was fired, and it reached at the tail of the pipeline. It usually means the last handler in the pipeline did not handle the exception.
io.netty.handler.codec.DecoderException: io.netty.handler.ssl.NotSslRecordException: not an SSL/TLS record: 474554202f76312f6170692f73736f2f76616c696461746520485454502f312e310d0a557365722d4167656e743a20506f73746d616e52756e74696d652f372e32362e380d0a4163636570743a202a2f2a0d0a43616368652d436f6e74726f6c3a206e6f2d63616368650d0a506f73746d616e2d546f6b656e3a2064316262626637652d303732352d346535362d623631322d3030343030306636316635310d0a486f73743a206c6f63616c686f73743a373030300d0a4163636570742d456e636f64696e673a20677a69702c206465666c6174652c2062720d0a436f6e6e656374696f6e3a206b6565702d616c6976650d0a0d0a
	at io.netty.handler.codec.ByteToMessageDecoder.callDecode(ByteToMessageDecoder.java:459)
	at io.netty.handler.codec.ByteToMessageDecoder.channelRead(ByteToMessageDecoder.java:265)
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:362)
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:348)
	at io.netty.channel.AbstractChannelHandlerContext.fireChannelRead(AbstractChannelHandlerContext.java:340)
	at io.netty.channel.DefaultChannelPipeline$HeadContext.channelRead(DefaultChannelPipeline.java:1359)
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:362)
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:348)
	at io.netty.channel.DefaultChannelPipeline.fireChannelRead(DefaultChannelPipeline.java:935)
	at io.netty.channel.nio.AbstractNioByteChannel$NioByteUnsafe.read(AbstractNioByteChannel.java:134)
	at io.netty.channel.nio.NioEventLoop.processSelectedKey(NioEventLoop.java:645)
	at io.netty.channel.nio.NioEventLoop.processSelectedKeysOptimized(NioEventLoop.java:580)
	at io.netty.channel.nio.NioEventLoop.processSelectedKeys(NioEventLoop.java:497)
	at io.netty.channel.nio.NioEventLoop.run(NioEventLoop.java:459)
	at io.netty.util.concurrent.SingleThreadEventExecutor$5.run(SingleThreadEventExecutor.java:858)
	at java.lang.Thread.run(Unknown Source)
Caused by: io.netty.handler.ssl.NotSslRecordException: not an SSL/TLS record: 474554202f76312f6170692f73736f2f76616c696461746520485454502f312e310d0a557365722d4167656e743a20506f73746d616e52756e74696d652f372e32362e380d0a4163636570743a202a2f2a0d0a43616368652d436f6e74726f6c3a206e6f2d63616368650d0a506f73746d616e2d546f6b656e3a2064316262626637652d303732352d346535362d623631322d3030343030306636316635310d0a486f73743a206c6f63616c686f73743a373030300d0a4163636570742d456e636f64696e673a20677a69702c206465666c6174652c2062720d0a436f6e6e656374696f6e3a206b6565702d616c6976650d0a0d0a
	at io.netty.handler.ssl.SslHandler.decodeJdkCompatible(SslHandler.java:1106)
	at io.netty.handler.ssl.SslHandler.decode(SslHandler.java:1162)
	at io.netty.handler.codec.ByteToMessageDecoder.decodeRemovalReentryProtection(ByteToMessageDecoder.java:489)
	at io.netty.handler.codec.ByteToMessageDecoder.callDecode(ByteToMessageDecoder.java:428)
	... 15 common frames omitted
22:42:21.086 INFO  vert.x-eventloop-thread-0 BaseServiceProxy     : -> GET /v1/api/iserver/auth/status,200|340ms
22:42:21.086 DEBUG vert.x-eventloop-thread-0 ClientPortalService  : status {"authenticated":false,"competing":false,"connected":true,"message":"","MAC":"F4:03:43:DB:84:00","fail":""}
22:42:21.086 INFO  vert.x-eventloop-thread-0 ClientPortalService  : process auth status {"authenticated":false,"competing":false,"connected":true,"message":"","MAC":"F4:03:43:DB:84:00","fail":""}
22:42:21.102 DEBUG vert.x-eventloop-thread-0 ClientPortalService  : auth brokerage, k=529d1942780228c4c9bbb8c501f196f2a97ec602, post={"username":"tymraf123","machineId":"23e586b2","compete":true}
22:42:21.176 INFO  vert.x-eventloop-thread-0 BaseServiceProxy     : -> GET /v1/api/iserver/auth/status,200|150ms
22:42:21.176 DEBUG vert.x-eventloop-thread-0 ClientPortalService  : status {"authenticated":false,"competing":false,"connected":true,"message":"","MAC":"F4:03:43:DB:84:00","fail":""}
22:42:21.177 INFO  vert.x-eventloop-thread-0 ClientPortalService  : process auth status {"authenticated":false,"competing":false,"connected":true,"message":"","MAC":"F4:03:43:DB:84:00","fail":""}
22:42:21.177 INFO  vert.x-eventloop-thread-0 ClientPortalService  : chanllenge already sent true
22:42:21.437 INFO  vert.x-eventloop-thread-0 BaseServiceProxy     : -> POST https://api.ibkr.com/v1/api/iserver/auth/ssodh/init,200|334ms
22:42:21.438 DEBUG vert.x-eventloop-thread-0 ClientPortalService  : iserver init : {"challenge":"dc7be5f7497327ef93f0b8cdf3bf965a683a570"}
22:42:21.438 INFO  vert.x-eventloop-thread-0 ClientPortalService  : sending sso dh response, k=e1b1cc0c0e7c96af2ebe39d662d2da63206c522e
22:42:21.583 INFO  vert.x-eventloop-thread-0 BaseServiceProxy     : -> POST https://api.ibkr.com/v1/api/iserver/auth/ssodh/response,200|144ms
22:42:21.583 INFO  vert.x-eventloop-thread-0 ClientPortalService  : server sso dh response {"error":"unknown"}
22:42:21.583 INFO  vert.x-eventloop-thread-0 ClientPortalService  : process auth status {"error":"unknown"}
22:42:21.583 DEBUG vert.x-eventloop-thread-0 ClientPortalService  : auth brokerage, k=529d1942780228c4c9bbb8c501f196f2a97ec602, post={"username":"tymraf123","machineId":"06a4844e","compete":true}
22:42:21.923 INFO  vert.x-eventloop-thread-0 BaseServiceProxy     : -> POST https://api.ibkr.com/v1/api/iserver/auth/ssodh/init,200|340ms
22:42:21.923 DEBUG vert.x-eventloop-thread-0 ClientPortalService  : iserver init : {"challenge":"dc7be5f7497327ef93f0b8cdf3bf965a683a570"}
22:42:21.923 INFO  vert.x-eventloop-thread-0 ClientPortalService  : sending sso dh response, k=e1b1cc0c0e7c96af2ebe39d662d2da63206c522e
22:42:22.067 INFO  vert.x-eventloop-thread-0 BaseServiceProxy     : -> POST https://api.ibkr.com/v1/api/iserver/auth/ssodh/response,200|144ms
22:42:22.067 INFO  vert.x-eventloop-thread-0 ClientPortalService  : server sso dh response {"error":"unknown"}
22:42:22.067 INFO  vert.x-eventloop-thread-0 ClientPortalService  : process auth status {"error":"unknown"}
22:42:22.067 DEBUG vert.x-eventloop-thread-0 ClientPortalService  : auth brokerage, k=529d1942780228c4c9bbb8c501f196f2a97ec602, post={"username":"tymraf123","machineId":"3b4e4eab","compete":true}
22:42:22.404 INFO  vert.x-eventloop-thread-0 BaseServiceProxy     : -> POST https://api.ibkr.com/v1/api/iserver/auth/ssodh/init,200|337ms
22:42:22.404 DEBUG vert.x-eventloop-thread-0 ClientPortalService  : iserver init : {"challenge":"dc7be5f7497327ef93f0b8cdf3bf965a683a570"}
22:42:22.404 INFO  vert.x-eventloop-thread-0 ClientPortalService  : sending sso dh response, k=e1b1cc0c0e7c96af2ebe39d662d2da63206c522e
22:42:22.549 INFO  vert.x-eventloop-thread-0 BaseServiceProxy     : -> POST https://api.ibkr.com/v1/api/iserver/auth/ssodh/response,200|145ms
22:42:22.549 INFO  vert.x-eventloop-thread-0 ClientPortalService  : server sso dh response {"error":"unknown"}
22:42:22.549 INFO  vert.x-eventloop-thread-0 ClientPortalService  : process auth status {"error":"unknown"}
22:42:22.549 DEBUG vert.x-eventloop-thread-0 ClientPortalService  : auth brokerage, k=529d1942780228c4c9bbb8c501f196f2a97ec602, post={"username":"tymraf123","machineId":"18a5a114","compete":true}
22:42:22.885 INFO  vert.x-eventloop-thread-0 BaseServiceProxy     : -> POST https://api.ibkr.com/v1/api/iserver/auth/ssodh/init,200|336ms
22:42:22.885 DEBUG vert.x-eventloop-thread-0 ClientPortalService  : iserver init : {"challenge":"dc7be5f7497327ef93f0b8cdf3bf965a683a570"}
22:42:22.885 INFO  vert.x-eventloop-thread-0 ClientPortalService  : sending sso dh response, k=e1b1cc0c0e7c96af2ebe39d662d2da63206c522e
22:42:23.035 INFO  vert.x-eventloop-thread-0 BaseServiceProxy     : -> POST https://api.ibkr.com/v1/api/iserver/auth/ssodh/response,200|150ms
22:42:23.036 INFO  vert.x-eventloop-thread-0 ClientPortalService  : server sso dh response {"error":"unknown"}
22:42:23.036 INFO  vert.x-eventloop-thread-0 ClientPortalService  : process auth status {"error":"unknown"}
22:42:23.036 DEBUG vert.x-eventloop-thread-0 ClientPortalService  : auth brokerage, k=529d1942780228c4c9bbb8c501f196f2a97ec602, post={"username":"tymraf123","machineId":"1155bb24","compete":true}
22:42:23.374 INFO  vert.x-eventloop-thread-0 BaseServiceProxy     : -> POST https://api.ibkr.com/v1/api/iserver/auth/ssodh/init,200|338ms
22:42:23.375 DEBUG vert.x-eventloop-thread-0 ClientPortalService  : iserver init : {"challenge":"dc7be5f7497327ef93f0b8cdf3bf965a683a570"}
22:42:23.375 INFO  vert.x-eventloop-thread-0 ClientPortalService  : sending sso dh response, k=e1b1cc0c0e7c96af2ebe39d662d2da63206c522e
22:42:23.519 INFO  vert.x-eventloop-thread-0 BaseServiceProxy     : -> POST https://api.ibkr.com/v1/api/iserver/auth/ssodh/response,200|144ms
22:42:23.519 INFO  vert.x-eventloop-thread-0 ClientPortalService  : server sso dh response {"error":"unknown"}
22:42:23.519 INFO  vert.x-eventloop-thread-0 ClientPortalService  : process auth status {"error":"unknown"}
22:42:23.519 DEBUG vert.x-eventloop-thread-0 ClientPortalService  : auth brokerage, k=529d1942780228c4c9bbb8c501f196f2a97ec602, post={"username":"tymraf123","machineId":"0aafb447","compete":true}
22:42:23.856 INFO  vert.x-eventloop-thread-0 BaseServiceProxy     : -> POST https://api.ibkr.com/v1/api/iserver/auth/ssodh/init,200|337ms
22:42:23.857 DEBUG vert.x-eventloop-thread-0 ClientPortalService  : iserver init : {"challenge":"dc7be5f7497327ef93f0b8cdf3bf965a683a570"}
22:42:23.857 INFO  vert.x-eventloop-thread-0 ClientPortalService  : sending sso dh response, k=e1b1cc0c0e7c96af2ebe39d662d2da63206c522e
22:42:24.007 INFO  vert.x-eventloop-thread-0 BaseServiceProxy     : -> POST https://api.ibkr.com/v1/api/iserver/auth/ssodh/response,200|150ms
22:42:24.007 INFO  vert.x-eventloop-thread-0 ClientPortalService  : server sso dh response {"error":"unknown"}
22:42:24.007 INFO  vert.x-eventloop-thread-0 ClientPortalService  : process auth status {"error":"unknown"}
22:42:24.007 DEBUG vert.x-eventloop-thread-0 ClientPortalService  : auth brokerage, k=529d1942780228c4c9bbb8c501f196f2a97ec602, post={"username":"tymraf123","machineId":"223abe84","compete":true}
22:42:24.341 INFO  vert.x-eventloop-thread-0 BaseServiceProxy     : -> POST https://api.ibkr.com/v1/api/iserver/auth/ssodh/init,200|334ms
22:42:24.342 DEBUG vert.x-eventloop-thread-0 ClientPortalService  : iserver init : {"challenge":"dc7be5f7497327ef93f0b8cdf3bf965a683a570"}
22:42:24.342 INFO  vert.x-eventloop-thread-0 ClientPortalService  : sending sso dh response, k=e1b1cc0c0e7c96af2ebe39d662d2da63206c522e
22:42:24.343 INFO  vert.x-eventloop-thread-0 BaseServiceProxy     : -> GET /v1/api/sso/validate,200|311ms
22:42:24.496 INFO  vert.x-eventloop-thread-0 BaseServiceProxy     : -> POST https://api.ibkr.com/v1/api/iserver/auth/ssodh/response,500|154ms
22:42:24.496 WARN  vert.x-eventloop-thread-0 BaseServiceProxy     : failed https://api.ibkr.com/v1/api/iserver/auth/ssodh/response | reason <!DOCTYPE HTML PUBLIC "-//IETF//DTD HTML 2.0//EN">
<html><head>
<title>500 Internal Server Error</title>
</head><body>
<h1>Internal Server Error</h1>
<p>The server encountered an internal error or
misconfiguration and was unable to complete
your request.</p>
<p>Please contact the server administrator at 
 webmaster@interactivebrokers.com to inform them of the time this error occurred,
 and the actions you performed just before this error.</p>
<p>More information about this error may be available
in the server error log.</p>
<p>Additionally, a 500 Internal Server Error
error was encountered while trying to use an ErrorDocument to handle the request.</p>
</body></html>

22:42:24.496 INFO  vert.x-eventloop-thread-0 ClientPortalService  : server sso dh response null
22:42:24.496 ERROR vert.x-eventloop-thread-0 ClientPortalService  : sso sh response failed
io.vertx.core.impl.NoStackTraceThrowable: <!DOCTYPE HTML PUBLIC "-//IETF//DTD HTML 2.0//EN">
<html><head>
<title>500 Internal Server Error</title>
</head><body>
<h1>Internal Server Error</h1>
<p>The server encountered an internal error or
misconfiguration and was unable to complete
your request.</p>
<p>Please contact the server administrator at 
 webmaster@interactivebrokers.com to inform them of the time this error occurred,
 and the actions you performed just before this error.</p>
<p>More information about this error may be available
in the server error log.</p>
<p>Additionally, a 500 Internal Server Error
error was encountered while trying to use an ErrorDocument to handle the request.</p>
</body></html>

22:42:24.496 WARN  vert.x-eventloop-thread-0 BaseServiceProxy     : failed /v1/api/iserver/auth/ssodh/response | reason 500
22:42:24.496 INFO  vert.x-eventloop-thread-0 ClientPortalService  : server sso dh response null
22:42:24.496 ERROR vert.x-eventloop-thread-0 ClientPortalService  : sso sh response failed
io.vertx.core.impl.NoStackTraceThrowable: 500
22:42:33.669 INFO  vert.x-eventloop-thread-0 BaseServiceProxy     : -> GET /v1/api/iserver/accounts,401|567ms
22:43:05.413 INFO  vert.x-eventloop-thread-0 BaseServiceProxy     : -> GET /v1/api/portfolio/accounts,200|1192ms
